README.md

```diff
@@ -11,16 +11,15 @@
 Easily install and use genomes in Python and elsewhere!
 
 The goal is to have a _simple_ and _straightforward_ way to download and use genomic sequences. 
-Currently, genomepy supports UCSC, Ensembl and NCBI. 
+Currently, Genomepy supports UCSC, Ensembl and NCBI. 
```

Comment:
I actually like the module lower-case. That means the package name and the "tool" name are identitical.